internal/conf/configuration.go

```diff
@@ -131,6 +131,41 @@ func (c *SessionsConfiguration) Validate() error {
 	return nil
 }
 
+type PasswordRequiredCharacters []string
+
+func (v *PasswordRequiredCharacters) Decode(value string) error {
+	parts := strings.Split(value, ":")
+
+	for i := 0; i < len(parts)-1; i += 1 {
```

Comment:
you'll miss the last character in the string 
```suggestion
	for i := 0; i <= len(parts)-1; i += 1 {
```

Reply:
Actually this is intentional, the last "part" is always going to be the proper set. It also works because of the `i+1` logic below, which would panic.

Reply:
The tests should handle all edge cases so I think it's safe.